src/nxdoc/web-ui/web-ui-functional-testing.md

```diff
@@ -80,8 +80,8 @@ To use the framework on custom projects, you need to create an npm package with
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@nuxeo/nuxeo-web-ui-ftest": "3.0.0-SNAPSHOT",
-    "cucumber": "^4.1.0"
+    "@nuxeo/nuxeo-web-ui-ftest": "file:./packages/nuxeo-web-ui-ftest",
```

Comment:
There won't be a "./packages/nuxeo-web-ui-ftest", unless you are working on web ui, right? If so, it won't work for custom projects. I would expect it to be something like:
```suggestion
    "@nuxeo/nuxeo-web-ui-ftest": "~3.0.0",
```